styles.css

```diff
@@ -0,0 +1,98 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+  list-style: none;
+  border: none;
+  text-decoration: none;
+}
+
+.navbar {
+  background: #ffff;
+  padding-top: 14px;
+  padding-bottom: 14px;
+  padding-left: 24px;
+  padding-right: 24px;
+  display: flex;
+  place-content: space-between;
+  position: relative;
+}
+
+.portfolio {
+  background-image: url("./img/mobile\ background.png");
+  background-repeat: no-repeat, no-repeat;
+  background-size: cover;
+  background-color: white;
+  border-radius: 0 0 0 100px;
+  padding-top: 114px;
+  padding-left: 24px;
+  padding-right: 24px;
+  height: 100vh;
+}
+
+.portfolio .intro {
+  position: static;
+  width: 600px;
+  left: 24px;
+  top: 114px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  color: #172b4d;
+  order: 0;
+  flex-grow: 0;
+  margin: 12px 0;
+  align-items: center;
+}
+
+.portfolio .more-intro {
+  font-weight: 300;
+  font-size: 16px;
+  line-height: 24px;
+  position: static;
+  width: 327px;
+  height: 146px;
+  left: 24px;
+  top: 276px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  color: #344563;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+  margin: 12px 0;
+}
+
+.portfolio .socials {
+  display: flex;
+  left: 0;
+  top: 0;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  color: #7f8cff;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin: 0;
+}
+
+.logo {
+  font-family: "Poppins", sans-serif;
+  font-size: 20px;
+  padding-top: 10px;
+  font-weight: bolder;
+}
```

Comment:
- [ ] Please add the color property on your logo here in order to match what we have on the Figma design that you chose. As a developer, paying attention to all details even the small ones will make you outstanding 😃 😎 
**HINT 💡** You may consider adding `color: #6070FF;` and check the output 😉

Reply:
Rectified